Tests/Features/ReachabilityConditionTests.swift

```diff
@@ -98,6 +98,17 @@ class ReachabilityConditionTests: OperationTests {
 
         XCTAssertEqual(error, ReachabilityCondition.Error.NotReachableWithConnectivity(.ViaWiFi))
     }
+
+    func test__condition_succeeds_when_only_wifi_accepted_and_only_wifi_available() {
+        let operation = TestOperation()
+
```

Comment:
Looks like you need to attach the condition here right? I guess you're waiting for the other PR to be merged though?